pkg/datasources/v1/datasources.go

```diff
@@ -36,9 +36,6 @@ type DataSourceFuncDef interface {
 // interact with external systems. These get taken into used by the Evaluator.
 // Moreover, a data source must be able to validate an update to itself.
 type DataSource interface {
-	// Returns the registered name of the data source.
-	GetName() string
-
```

Comment:
Was this intentional or it's no longer needed?

Reply:
Intentional https://github.com/mindersec/minder/pull/5009/commits/2bab631b2af768ecb1ec51a386da87888ac6e4ee

data source drivers no longer have access to their own name, and all of this is now tracked by the data source registry. It was a leftover from the initial interface drafting.

Reply:
Got it 👍